packages/crud/src/crud/CrudTableHeader.tsx

```diff
@@ -174,7 +202,7 @@ const CrudTableHeader: React.FC<CrudTableHeaderProps> = (props) => {
           <Box
             sx={{
               width: '100%',
-              maxWidth: { xs: '100%', md: styleConfig.searchWidth },
+              maxWidth: { xs: '100%', md: styleConfig.searchWidth }
```

Comment:
This is strange. We expect commas here because this is an object. There should be dangling commas rule applied here